src/ray/gcs/tables.cc

```diff
@@ -63,7 +68,12 @@ Status Log<ID, Data>::AppendAt(const JobID &job_id, const ID &id,
                                std::shared_ptr<DataT> &dataT, const WriteCallback &done,
                                const WriteCallback &failure, int log_length) {
   num_appends_++;
-  auto callback = [this, id, dataT, done, failure](const std::string &data) {
+  const auto start_time = current_time_ms();
+  auto callback = [this, id, dataT, done, failure, start_time](const std::string &data) {
+    const auto end_time = current_time_ms();
+    stats::RedisLatency().Record(
+        end_time - start_time,
+        {{stats::NodeAddressKey, RayConfig::instance().node_address()}});
```

Comment:
Should these sort of global tags be auto added?

Reply:
Thanks, I'd like to address this in next PR which aims to address `node_address` field issue in `RayConfig`(I left a TODO there).